cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -175,4 +176,7 @@ class UploadResultToSw360Command : CliktCommand(
             // Upload the uncurated packages because SW360 also is a package curation provider.
             project.collectDependencies().mapNotNull { ortResult.getUncuratedPackageById(it) }
         }
+
+    private fun createReleaseName(pkgId: Identifier) =
+        listOfNotNull(pkgId.namespace, pkgId.name).joinToString("/")
```

Comment:
None of the used properties is nullable, so there should be no need to use `listOfNotNull()` (even if this was already the case in the original code). Also, in this simply case, I'd just use this one-liner:

    private fun createReleaseName(pkgId: Identifier) = "${pkgId.namespace}/${pkgId.name}"

Reply:
Now I notice another problem, if none of the properties are nullable. If namespace is empty, it would create an release name with a leading slash: `/example-package`

Could i use `listOf(pkgId.namespace, pkgId.name).filter { it.isNotEmpty() }.joinToString("/")` or just deal with the leading slash in release name to point out that there is no namespace?

Reply:
I basically like your proposed `listOf()` approach. But I'm still wondering what the conventions for a release name in SW360 are. E.g., is it really supposed to *not* contain version? Because if it should / may contain a version, we could again take a look at `Identifier.toPath()`. But if only *namespace* and *name* should be part of a release name in SW360, then I'd go with your `listOf()` code.

Reply:
Yes, I think it shouldn't contain the version in the release name, because there is a specified field for the version.